rat/fit.py

```diff
@@ -57,7 +58,7 @@ def _build_constrained_dfs(
         df = pandas.concat([base_df] * num_draws * num_chains, ignore_index=True)
         df["chain"] = numpy.repeat(numpy.arange(num_chains), size * num_draws)
         df["draw"] = numpy.tile(numpy.repeat(numpy.arange(num_draws), size), num_chains)
-        df["value"] = constrained_variable.flatten(order="C")
+        df[name] = constrained_variable.flatten(order="C")
```

Comment:
It was convenient for the MRP example to have variables named by their actual name instead of value (so easier to join multiple tables together). So I changed this everywhere.